src/sql/workbench/parts/tasks/browser/tasksRenderer.ts

```diff
@@ -36,15 +39,15 @@ export class TaskHistoryRenderer implements IRenderer {
 	/**
 	 * Returns the element's height in the tree, in pixels.
 	 */
-	public getHeight(tree: ITree, element: any): number {
+	public getHeight(tree: ITree, element: TaskNode): number {
 		return TaskHistoryRenderer.TASKOBJECT_HEIGHT;
 	}
 
 	/**
 	 * Returns a template ID for a given element.
 	 */
-	public getTemplateId(tree: ITree, element: any): string {
-		return TaskHistoryRenderer.TASKOBJECT_TEMPLATE_ID;
+	public getTemplateId(tree: ITree, element: TaskNode): string {
+		return element.id;
```

Comment:
This is unfortunately an inefficiency, but is required based on what we are trying to accomplish and how vscode's tree works.